.storybook/decorators/GeneratedStory/GeneratedStory.styles.ts

```diff
@@ -0,0 +1,62 @@
+import { css } from '@leafygreen-ui/emotion';
```

Comment:
Should we be committing generated files?

Reply:
No?